indexed-file-analyzer/src/main/kotlin/com/avail/tools/fileanalyzer/IndexedFileAnalyzer.kt

```diff
@@ -310,11 +309,65 @@ object IndexedFileAnalyzer
 			// and ensure that any supplied paths are syntactically valid.
 			configuration = configure(args)
 			with(configuration) {
+				if (implodeDirectory !== null)
+				{
+					assert(inputFile === null)
+					assert(implodeHeader !== null)
+					assert(implodeOutput !== null)
+					val dir = implodeDirectory!!
+					// The default option file is used as the output.
+					val outputFile = implodeOutput!!
+					if (outputFile.exists())
+					{
+						throw Exception(
+							"Output file must not already exist")
+					}
+					val files = dir.list()!!.toMutableList()
+					val metadata = when
+					{
+						files.remove("metadata") -> "metadata"
+						files.remove("metadata.txt") -> "metadata.txt"
+						else -> null
+					}
```

Comment:
We should prevent both `metadata` and `metadata.txt` from both being present. The code here gives `metadata` priority and ignores `metadata.txt`, but that is undesired behavior.

Reply:
Done.